### PR TITLE
Hand platform

### PR DESCRIPTION
**Pull request by Jlrine2**

### CODE REVIEW

File: entry.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash --login
+set -e
+conda activate asf-tools
+exec calculate_hand "$@"
```
Comment (jhkennedy) [inline entry.sh:4]: Did you change the `make_hand` to `calculate_hand`? It's still `make_hand` in `setup.py`:
https://github.com/ASFHyP3/asf-tools/blob/hand-platform/setup.py#L63

Merge conflict resolution problem?

File: asf_tools/hand.py [hunk 2]
```diff
@@ -1,13 +1,15 @@
 """Calculate Height Above Nearest Drainage (HAND) from the Copernicus GLO-30 Public DEM"""
 import argparse
 import logging
+import os
```
Comment (Jlrine2) [inline asf_tools/hand.py:4]:
```suggestion
import something else
```

Comment (Jlrine2) [inline asf_tools/hand.py:4]:
```suggestion
import os1
```
```suggestion
import os2
```

**Comment by jhkennedy** [comment]
Closing this since we're doing with global campaign -- if we need do another campaign, we can re-open.

**Pull request closed without merging**